job/callback_info.go

```diff
@@ -0,0 +1,33 @@
+package job
+
+import "fmt"
+
+// CallbackInfo holds info to be posted back to the provided callback url.
+type CallbackInfo struct {
+	Success      bool   `json:"success"`
+	Error        string `json:"error"`
+	Extra        string `json:"extra"`
+	ResourceURL  string `json:"resource_url"`
+	DownloadURL  string `json:"download_url"`
+	JobID        string `json:"job_id"`
+	ResponseCode int    `json:"response_code"`
+}
+
+// NewCallbackInfo creates a callback info object
+func NewCallbackInfo(success bool, err, extra, resourceURL, downloadURL, jobID string, responseCode int) CallbackInfo {
```

Comment:
Having a constructor when all the fields as needed makes no much sense. In this case, you just do a `CallbackInfo{.....}`.